vm-setup/roles/v1a2_integration_test/tasks/provision.yml

```
@@ -0,0 +1,104 @@
+---
+# Populate crs with environment variables
+  - name: Template cluster.yaml
+    template:
+      src: "{{ HOME }}/metal3-dev-env/vm-setup/roles/v1a2_integration_test/crs/cluster.yaml"
+      dest: "/tmp/cluster.yaml"
+    ignore_errors: true
+
+  - name: Template controlplane_ubuntu.yaml
+    template:
+      src: "{{ HOME }}/metal3-dev-env/vm-setup/roles/v1a2_integration_test/crs/controlplane_ubuntu.yaml"
+      dest: "/tmp/controlplane_ubuntu.yaml"
+    when: IMAGE_OS == "Ubuntu"
+    ignore_errors: true
+
+  - name: Template workers_ubuntu.yaml
+    template:
+      src: "{{ HOME }}/metal3-dev-env/vm-setup/roles/v1a2_integration_test/crs/workers_ubuntu.yaml"
+      dest: "/tmp/workers_ubuntu.yaml"
+    when: IMAGE_OS == "Ubuntu"
+    ignore_errors: true
+
+  - name: Template controlplane_centos.yaml
+    template:
+      src: "{{ HOME }}/metal3-dev-env/vm-setup/roles/v1a2_integration_test/crs/controlplane_centos.yaml"
+      dest: "/tmp/controlplane_centos.yaml"
+    vars:
+      machine_env:
+        IMAGE_NAME: '{{ IMAGE_NAME_CENTOS }}'
+        IMAGE_LOCATION: '{{ IMAGE_LOCATION_CENTOS }}'
+        IMAGE_USERNAME: '{{ IMAGE_USERNAME_CENTOS }}'
+    when: (IMAGE_OS == "Centos") or
+          (IMAGE_OS == "")
+    ignore_errors: true
+
+  - name: Template workers_centos.yaml
+    template:
+      src: "{{ HOME }}/metal3-dev-env/vm-setup/roles/v1a2_integration_test/crs/workers_centos.yaml"
+      dest: "/tmp/workers_centos.yaml"
+    vars:
+      machine_env:
+        IMAGE_NAME: '{{ IMAGE_NAME_CENTOS }}'
+        IMAGE_LOCATION: '{{ IMAGE_LOCATION_CENTOS }}'
+        IMAGE_USERNAME: '{{ IMAGE_USERNAME_CENTOS }}'
+    when: (IMAGE_OS == "Centos") or
+          (IMAGE_OS == "")
+    ignore_errors: true
+
+# Provision cluster & machines
+  - name: Create a cluster
+    k8s:
+      state: present
+      src: "/tmp/cluster.yaml"
+      namespace: "{{ NAMESPACE }}"
+      ignore_errors: true
```

you should not ignore the error, rather specify what you expect